server/server.go

```diff
@@ -374,6 +374,11 @@ func (s *Server) StartExitMonitor() {
 	}
 	defer watcher.Close()
 
+	containerStoppedJobs := make(chan string, 100)
+	for w := 1; w <= 10; w++ {
```

Comment:
if the 10 here is what you were talking about putting into crio.conf 👍   Otherwise please consider it.